Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -535,6 +535,23 @@ void ChessWidget::playback_move(PlaybackDirection direction)
     update();
 }
 
+void ChessWidget::update_move_display_widget(Chess::Board& board)
+{
+    size_t i = 1;
+    size_t turn = 1;
+    StringBuilder sb;
+    for (auto const& move : board.moves()) {
```

Comment:
I think there's `AK/Enumerate.h` now, so this could be:s

```suggestion
    for (auto [i, move] : enumerate(board.moves())) {
```

Reply:
Oh cool, good to know! I'll make that change